applications/nrf_desktop/Kconfig.hid

```diff
@@ -179,4 +179,45 @@ endchoice
 
 endmenu
 
+menu "HID device identification"
+
+config DESKTOP_DEVICE_PRODUCT
+	string "Product name"
+	default "Dongle nRF52 Desktop" if DESKTOP_ROLE_HID_DONGLE
+	default "Mouse nRF52 Desktop" if DESKTOP_PERIPHERAL_TYPE_MOUSE
+	default "Keyboard nRF52 Desktop" if DESKTOP_PERIPHERAL_TYPE_KEYBOARD
+	default "HID nRF52 Desktop"
```

Comment:
I wonder if we shouldn't maybe add `&& SOC_SERIES_NRF52X` to `if`s, as defaults implicate that those devices are based on nRF52 series?

Reply:
I wanted to change `nRF52` to `nRF` to avoid depending on SoC family. Unfortunately we already need to maintain backwards compatibility of the application and changing name here could actually break it . nRF Desktop dongles filter advertising devices relying on Bluetooth device name which is also `DEVICE_TYPE nRF52 Desktop`, so I think it may be safer to keep it as it is (since I plan to use the product name also for Bluetooth advertising).

Reply:
Ok. I agree, there are too many risks. Lets leave it like it is.